salt/elife-dashboard/scheduler.sls

```diff
@@ -102,31 +107,32 @@ configure-{{ app.name }}-log:
         - require:
             - install-{{ app.name }}
 
-# temporary state. 
-old-uwsgi-{{ app.name }}:
-    file.absent:
-        - name: /etc/init.d/uwsgi-{{ app.name }}
-
-uwsgi-{{ app.name }}:
+uwsgi-elife-article-scheduler-upstart:
     file.managed:
         - name: /etc/init/uwsgi-{{ app.name }}.conf
         - source: salt://elife-dashboard/config/etc-init-uwsgi-elife-article-scheduler.conf
         - template: jinja
         - mode: 755
 
+uwsgi-elife-article-scheduler-systemd:
+    file.managed:
+        - name: /lib/systemd/system/uwsgi-{{ app.name }}.service
+        - source: salt://elife-dashboard/config/lib-systemd-system-uwsgi-elife-article-scheduler.service
+        - template: jinja
+
+uwsgi-{{ app.name }}:
     service.running:
         - enable: True
         - require:
-            - file: old-uwsgi-{{ app.name }}
-            - file: uwsgi-params
-            - pip: uwsgi-pkg
-            - file: uwsgi-app            
-            - file: app-uwsgi-conf
-            - file: app-nginx-conf
-            - file: app-log-file
+            - uwsgi-elife-article-scheduler-upstart
```

Comment:
I found it strange all these were broken personally. These are requisites for the other service to run and is probably a copy+paste job.